packages/osd-plugin-helpers/src/integration_tests/build.test.ts

```diff
@@ -193,8 +195,10 @@ it('builds a non-semver generated plugin into a viable archive', async () => {
      info running @osd/optimizer
      │ info initialized, 0 bundles cached
      │ info starting worker [1 bundle]
-     │ warn worker stderr Browserslist: caniuse-lite is outdated. Please run:
-     │ warn worker stderr npx browserslist@latest --update-db
+     │ warn worker stderr You did not set any plugins, parser, or stringifier. Right now, PostCSS does nothing. Pick plugins for your case on https://www.postcss.parts/ and use them in postcss.config.js.
+     │ warn worker stderr You did not set any plugins, parser, or stringifier. Right now, PostCSS does nothing. Pick plugins for your case on https://www.postcss.parts/ and use them in postcss.config.js.
+     │ warn worker stderr You did not set any plugins, parser, or stringifier. Right now, PostCSS does nothing. Pick plugins for your case on https://www.postcss.parts/ and use them in postcss.config.js.
+     │ warn worker stderr You did not set any plugins, parser, or stringifier. Right now, PostCSS does nothing. Pick plugins for your case on https://www.postcss.parts/ and use them in postcss.config.js.
```

Comment:
Why is this repeated so many times? Is this auto generated?

Reply:
I think this is related to the postcss change?

Reply:
Yeah it is auto generated.

Reply:
The repeated messages we're seeing (│ warn worker stderr You did not set any plugins, parser, or stringifier. Right now, PostCSS does nothing. Pick plugins for your case on https://www.postcss.parts/ and use them in postcss.config.js.) are warning messages. They come up multiple times because, during the build process, there might be multiple instances or modules where PostCSS is invoked without being properly configured. It's warning you each time it encounters such a situation.

The reason for disable `autoprefixer` in `postcss` and cause it not work could refer to this explain https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4649/files#r1280889212